test/support.js

```diff
@@ -12,6 +12,9 @@ TaskEmitter = require('strong-task-emitter');
 request = require('supertest');
 var RemoteObjects = require('strong-remoting');
 
+// Use local registries
+loopback.localRegistry = true;
```

Comment:
If this PR is not going to change Gruntfile to run the tests twice, then this change must be reverted together with other places where you manipulate `loopback.localRegistry` in before/after hooks.

The idea is to run existing unit-tests with global registry to ensure we are not introducing any regression in the existing applications.